### ISSUE TITLE
new copy in VEC repo needs to be integrated

### ISSUE THREAD

**Issue by emilyville** [department-of-veterans-affairs/vets-website#1546]
I'm not sure who's got the ball on this one, but we need to better integrate this copy. As is, it doesn't "replace" what's there enough to remove anything, and often puts the tool so far down on the page that it's hard to find. 

Tagging @gnakm @rcarthur and assigning to @mariana-almeida 

Search Page
![search](https://cloud.githubusercontent.com/assets/1307774/14068497/e6071eac-f439-11e5-9a7a-c0d41f518a89.png)

Resume Builder
![screen shot 2016-03-27 at 4 10 16 pm](https://cloud.githubusercontent.com/assets/1307774/14068499/e871f608-f439-11e5-8c39-e8fc7ec2bbf6.png)

Skills Translator
![skills-translator](https://cloud.githubusercontent.com/assets/1307774/14068500/e9cd0a7e-f439-11e5-910f-1e56b8a5d294.png)

Hiring Commitment
![commitments](https://cloud.githubusercontent.com/assets/1307774/14068502/eb4d16fa-f439-11e5-8af6-e8147c1abbd9.png)

**Comment by gnakm** [comment]
I always loved the way gov.uk tackled this https://www.gov.uk/jobsearch#before-you-start
They have a 1 or 2 liner description, the tool and all the "what you need to know" or other information below. @rcarthur Would it be possible to condense the copy even more?

This is an example just for the job search page but the idea is the have to tool on top/middle of the page and the copy integrated below. 

example with accordian
![accordian](https://cloud.githubusercontent.com/assets/13420618/14080170/4dd33646-f4d1-11e5-81ad-0be6cf855df7.png)

example with tabs
![tabs](https://cloud.githubusercontent.com/assets/13420618/14080173/50f91c0a-f4d1-11e5-8647-f86163a4f61b.png)

Also I kept the widgets kind of vague based off the [... widget in the sidebar] notes on the screenshots.

**Comment by rcarthur** [comment]
Yes, agree 100%.  The content that we developed for these pages was written without knowing exactly what the pages would look like.  

Seems like there are 3 places where we can put content on these pages. 

1)  2 sentences above the tool. 
2)  A couple nuggets (alongside the too)
3)  Added context at the bottom of the tool.  

What is the best way for us to proceed here?  Could we set up a 20 minute call today to discuss how to sync up the intent of the new content with the design elements?

Thanks!!!!
